tests/terraform_compliance/steps/its_key_value_test_cases/root_and_values_1.json

```diff
@@ -0,0 +1,13 @@
+{
+    "stash": [
+        {
+            "id": "a",
+            "foo": 1000,
+            "values": {
+                "foo": 2000
+            }
+        }
+    ],
+    "args": ["foo", 1000],
+    "expected_ids": []
```

Comment:
Items inside `values` take precedence. Is that right?

Reply:
True